.changeset/silent-bananas-speak.md

```diff
@@ -0,0 +1,8 @@
+---
+'@leafygreen-ui/hooks': minor
+'@leafygreen-ui/text-input': patch
```

Comment:
We should probably have two changesets here given the changes are unrelated.

I believe this currently will generate a changelog for text input that includes the creation of `usePoller`.

Reply:
Oooh that's how it works? Interesting.